docs/docs/APIdocs/layers/nn/Pooling Layers/RoiPooling.md

```diff
@@ -0,0 +1,215 @@
+           -0.81542057  0.14578317      -0.6531974      0.60776395      -0.32058007     -1.80771481.7660322       1.0680646
+           1.1328241    0.43677545      -0.9402618      -1.3002211      0.26012567      1.69481340.37849447       0.39286092
+           1.9443163    0.5415504       1.0793099       1.3312546       0.48346 1.2019655       0.3718734 0.21091922
+           0.5499047    1.6418253       0.8064177       0.37626198      0.8736181       -0.40816033       -0.5806787      1.286581
+           -0.5904657   -0.21188398     -0.040509004    1.2989452       1.6827602       1.3229258-0.68433124      0.87974
+
+           (2,2,.,.) =
+           -0.09759476  -0.32767114     0.16223079      2.3114302       -0.48496276     1.19290720.8572289        0.43429425
+           -1.0245247   0.19002944      1.5659521       -1.3689835      -1.4437296      -0.38216656       0.6333655       -0.57124794
+           -0.31111157  1.5184602       -1.3835855      -0.9295573      2.244521        -1.11849820.5451996       -0.4441631
+           -1.534093    -0.5599659      1.1980947       -1.0140935      1.3288999       0.19487387-0.1261734      -1.2222558
+           -0.070535585 0.9047848       -0.6719811      -1.6532638      -0.5290511      -0.18300447       0.69385433      0.018756092
+           0.24767837   0.620484        -0.5346291      1.0685066       -0.36903372     -0.26955062       1.1042496       0.5944603
+
+           [com.intel.analytics.bigdl.tensor.DenseTensor$mcF$sp of size 2x2x6x8]
+ }
+ 
+scala> print(output)
+(1,1,.,.) =
+1.0994664       2.3391871       1.5409279
+1.3795123       1.3795123       0.6159163
+
+(1,2,.,.) =
+1.5452348       1.5452348       1.3372624
+0.8858275       0.9677718       1.5691053
+
+(2,1,.,.) =
+0.37849447      0.39286092      0.39286092
+-0.5806787      1.286581        1.286581
+
+(2,2,.,.) =
+0.5451996       0.5451996       -0.4441631
+1.1042496       1.1042496       0.5944603
+
+(3,1,.,.) =
+0.60776395      1.6948134       1.7660322
+1.3312546       1.2019655       1.2019655
+
+(3,2,.,.) =
+2.244521        2.244521        0.6333655
+1.3288999       1.3288999       0.69385433
+
+(4,1,.,.) =
+-0.40570387     -0.40570387     -0.40570387
+-0.40570387     -0.40570387     -0.40570387
+
+(4,2,.,.) =
+0.4595303       0.4595303       0.4595303
+0.4595303       0.4595303       0.4595303
+
+[com.intel.analytics.bigdl.tensor.DenseTensor of size 4x2x2x3]
+
+```
+
+**Python example:**
+```python
+from bigdl.nn.layer import *
+import numpy as np
+
+input_data = np.random.randn(2,2,6,8)
+input_rois = np.array([0, 0, 0, 7, 5, 1, 6, 2, 7, 5, 1, 3, 1, 6, 4, 0, 3, 3, 3, 3],dtype='float64').reshape(4,5)
+print "input is :",[input_data, input_rois]
+
+m = RoiPooling(3,2,1.0)
+out = m.forward([input_data,input_rois])
+print "output of m is :",out
+```
+produces output:
+
+```python
+input is : [array([[[[ 0.08500103,  0.33421796,  0.29084699,  1.60344635, -0.24289341,
```

Comment:
RoiPooling needs to specify regions of interest in the original picture. So we need a larger input to better demonstrate the roi regions. I used the same size as in unit test.